ara/templates/playbook.html

```diff
@@ -93,8 +106,9 @@
     <tr>
       <td>{{ task.offset_from_playbook |timefmt }}</td>
       <td>{{ task.duration |timefmt }}</td>
+      <td>{{ task.name }}</td>
       <td>{{ task.action }}</td>
-      <td>{{ task.path|pathtruncate }}</td>
+      <td>{{ task.file.path|pathtruncate }}</td>
       <td>{{ task.lineno }}</td>
```

Comment:
That is what I want to have, ultimately, but I was going to leave that for another pr, since I think that's going to be e more substantial chunk of UI work.